code/main/edge/edgeModule.brs

```diff
@@ -35,31 +35,27 @@ function _adb_EdgeModule(configurationModule as object, identityModule as object
         _identityModule: identityModule,
         _edgeRequestWorker: _adb_EdgeRequestWorker(),
 
-        processEvent: function(requestId as string, eventData as object, timestampInMillis as longinteger) as dynamic
+        ''' sendEvent API triggers this API to queue edge requests
+        ''' requestId: unique id for the request
+        ''' eventData: data to be sent to edge
+        ''' timestampInMillis: timestamp of the event
+        processEvent: function(requestId as string, eventData as object, timestampInMillis as longinteger) as void
```

Comment:
Can we rename this function as it only queues an event and doesn’t process it?

Reply:
I was also thinking os something similar but I was thinking to have same public APIs for each modules, since this is being called from processor, and it is upto the modules to how to process the event. In this case we are simply queuing. But having common methods across modules which get called from the processor should be our goal to have some structure.

How do you think about `handleEvent()` per module?

Reply:
Let's decide later after adding the Consent module.

Reply:
We can consider merging the processEvent and queueEdgeRequest functions during or after implementing consent, so that Edge only exposes a single public method.